src/main/java/org/gradlex/jvm/dependency/conflict/detection/rules/CapabilityDefinitionRule.java

```diff
@@ -54,5 +54,9 @@ protected String getVersion(ModuleVersionIdentifier id) {
         return id.getVersion();
     }
 
+    protected void additionalAdjustments(ModuleVersionIdentifier id, VariantMetadata variant) {
+        additionalAdjustments(variant);
+    }
```

Comment:
Should this replace the overloaded method? I don't thing this should be considered a public API so introducing a breaking change like that could be acceptable.